tests/unit/flow/test_flow_eval_pod.py

```diff
@@ -23,14 +25,14 @@ class DummyEvaluator3(DummyEvaluator1):
 docs = list(random_docs(1))
 
 
-def test_flow1():
+def test_flow1(tmpdir):
```

Comment:
tmpdir I think does nothing here. Because nothing forces the `BaseExecutor` to have `tmpdir` as workspace folder

Reply:
yes